custom-elements.json

```diff
@@ -0,0 +1,316 @@
+{
+  "schemaVersion": "1.0.0",
```

Comment:
Why is this file here, but also listed in .gitignore? :)

Reply:
no idea 😂 just googled this and it's likely because I pushed up the file before adding it to gitignore, so I just need to fix that in my local git history